deployment/GI02HYPM_Deploy.csv

```diff
@@ -44,3 +44,12 @@ AR30-03,,,,GI02HYPM-WFP02-04-CTDPFL000,5,1,2018-06-09T12:55:00,,CGMGI-02HYPM-000
 AR30-03,,,,GI02HYPM-WFP02-05-VEL3DL000,5,1,2018-06-09T12:55:00,,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.96974,-39.49005,,2573,2669,
 AR30-03,,,,GI02HYPM-MPM01-02-ZPLSGA009,5,1,2018-06-09T12:55:00,,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55109,59.96974,-39.49005,,,2669,upward-looking
 AR30-03,,,,GI02HYPM-MPM01-02-ZPLSGA010,5,1,2018-06-09T12:55:00,,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55111,59.96974,-39.49005,,,2669,downward-looking
+AR35-05,,,,GI02HYPM-RIM01-00-SIOENG000,6,1,,,CGMGI-02HYPM-00006,,CGCON-SIOCON-12368,,,,2585,2668,
+AR35-05,,,,GI02HYPM-RIM01-02-CTDMOG039,6,1,,,CGMGI-02HYPM-00006,,CGINS-CTDMOG-11611,,,,138,2668,
+AR35-05,,,,GI02HYPM-WFP02-00-WFPENG000,6,1,,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGCON-GWFPGX-77401,,,,2573,2668,
+AR35-05,,,,GI02HYPM-WFP02-01-FLORDL000,6,1,,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-FLORDL-02350,,,,2573,2668,
+AR35-05,,,,GI02HYPM-WFP02-03-DOSTAL000,6,1,,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,,,,2573,2668,
+AR35-05,,,,GI02HYPM-WFP02-04-CTDPFL000,6,1,,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,,,,2573,2668,
+AR35-05,,,,GI02HYPM-WFP02-05-VEL3DL000,6,1,,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,,,,2573,2668,
+AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA009,6,1,,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55069,,,,,2668,upward-looking
+AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA010,6,1,,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55066,,,,,2668,downward-looking
```

Comment:
Ref Des and S/Ns look good.  We don't need to put the deployment depths and water depths.  The water depths listed on are not design depths and are not close (e.g., 138 m for CTDMO vs. 160 m).  In the future we should leave them blank; we need to be sure to update them after deployment.